tests/publiccloud/prepare_tools.pm

```diff
@@ -81,11 +81,11 @@ sub run {
     assert_script_run("ipa list");
     assert_script_run("ipa --version");
 
-    # Download and Install Terraform
-    my $terraform_url = get_var('TERRAFORM_URL', 'https://releases.hashicorp.com/terraform/0.11.13/terraform_0.11.13_linux_amd64.zip');
-    assert_script_run("wget -q $terraform_url");
-    assert_script_run('unzip terraform_* terraform -d /usr/bin/');
-    assert_script_run('terraform -v');
+    # Install Terraform from repo
+    my $tfm_repo = 'https://download.opensuse.org/repositories/systemsmanagement:/terraform/' . generate_version() . '/systemsmanagement:terraform.repo';
```

Comment:
I would recommend to use  test suite variable TERRAFORM_REPO instead of hardcoded repository.

Reply:
ok

Reply:
disadvantage of variable that changes to value are out of git , in case there is not hard requirement to change variable runtime better to keep it how @jlausuch did

Reply:
If there will be another major terraform update, you can easily change variable to point to unstable repo. Hardcoded urls are not flexible and they don't have to be in git.

Reply:
I think to introducing a VAR would be nice. And use this `$tfm_repo` as default. So we could play around, if we run in trouble without touching code.